index.js

```diff
@@ -22,7 +22,7 @@ function stubMethod(service, method, replacement) {
   if (!isStubbed(service)) stubService(service);
   if (!replacement) return sinon.stub(getService(service).prototype, method);
 
-  return sinon.stub(getService(service).prototype, method, function(params, callback) {
+  return sinon.stub(getService(service).prototype, method).callsFake(function(params, callback) {
```

Comment:
Hopefully the message won't show up, since I've updated this module's usage of sinon stubs to use the .callsFake() method recommendation, which sinon is no longer printing. It keeps the API the same for this module, while updating the sinon API usage under the hood.